src/sentry/models/identity.py

```diff
@@ -31,15 +31,15 @@ class IdentityProvider(Model):
     __core__ = False
 
     type = models.CharField(max_length=64)
-    organization = FlexibleForeignKey('sentry.Organization')
+    organization_id = BoundedPositiveIntegerField(default=0)
```

Comment:
Should this be IntegerField?

Reply:
Nah, this is fine. I do like that South doesn't apply the migration to make `default=0`, but that's fine, I'll do it by hand.

Reply:
I had to change the south migration completely. Renaming a field is considered deleting and adding a column which wouldn't work for us. After I modified it I couldn't tell how to add the default constraint. Can we do it here if possible?

Reply:
I'd try to just change it on another column, and see what South generates, then steal that.